src/installers/index.ts

```diff
@@ -15,11 +15,12 @@ export const availablePackages = [
 
 export type AvailablePackages = typeof availablePackages[number];
 
-export type Installer = (
-  projectDir: string,
-  packageManager: PackageManager,
-  packages: PkgInstallerMap,
-) => Promise<void>;
+export type Installer = (opts: {
```

Comment:
Oh you have it here, maybe you could use it in the scaffoldProject and lognextstep